message_definitions/v1.0/loweheiser.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<mavlink>
+  <!-- Loweheiser contact info: -->
+  <!-- company URL: https://www.loweheiser.com/ -->
+  <!-- company LinkeIn: https://www.linkedin.com/company/loweheiser -->
+  <!-- email contact: info@loweheiser.com -->
+  <!-- mavlink ID range: 10150 - 10199 -->
+  <include>minimal.xml</include>
+  <enums>
+    <enum name="MAV_CMD">
+      <!-- Loweheiser specific MAV_CMD_* commands -->
+      <entry name="MAV_CMD_LOWEHEISER_SET_STATE" value="10151">
+        <description>Set Loweheiser desired states</description>
+        <param index="1">EFI Index</param>
+        <param index="2">Desired Engine/EFI State (0: Power Off, 1:Running)</param>
```

Comment:
```suggestion
        <param index="2">Desired Engine/EFI State (0: Power Off, 1:Run)</param>
```

Reply:
Mmm.. these are desired states, not commands, thus the adjective form.

Reply:
meh i wont fight ya